tools/preflight-check.py

```diff
@@ -17,30 +20,41 @@ def main():
                 continue
 
             if not row:
-                print("Empty row found:", i)
-                exit(1)
+                print(f"::error file={filename},line={i}::empty row found")
+                has_error = True
+                continue
+
+            if len(row) != 6:
+                print(f"::error file={filename},line={i}::incorrect number of columns")
+                has_error = True
+                continue
 
             title = row[0]
             store = row[1]
             codename = row[2]
             umu_id = row[3]
 
             if not (title and store and codename and umu_id):
-                print("At least one of the required fields is missing, in row:", i)
-                exit(1)
+                print(f"::error file={filename},line={i}::At least one of the required fields is missing")
+                has_error = True
+                continue
 
             if store not in SUPPORTED_STORES:
-                print("Invalid store provided", store, "in row", i)
-                exit(1)
+                print(f"::error file={filename},line={i}::Invalid store provided {store}")
+                has_error = True
+                continue
 
             if store == "none" and codename == "none":
                 continue
             release_id = f"{store}_{codename}"
             if release_id in release_ids:
-                print("Duplicate entry found", title, store, codename, "in row", i)
-                exit(1)
+                print(f"::error file={filename},line={i}::Duplicate entry found {title}, {store}, {codename}")
```

Comment:
Nit: Add quotes to the entry for readability.

`print(f"::error file={filename},line={i}::Duplicate entry found '{title}, {store}, {codename}'")`